src/site/stages/build/process-cms-exports/index.js

```diff
@@ -104,9 +73,6 @@ const entityAssemblerFactory = contentDir => {
 
     // Recursively expand entity references
     for (const [key, prop] of Object.entries(filteredEntity)) {
-      // eslint-disable-next-line no-continue
-      if (ignoreList.includes(key)) continue;
```

Comment:
This is where the magic happens.

By not continuing here, we'll try to expand everything that looks like an entity reference.